docs/source/users.rst

```diff
@@ -81,11 +83,12 @@ To create a new user, send a POST request to the global ``/@users`` endpoint wit
         'username': 'noam',
         'email': 'noam.chomsky@mit.edu',
         'password': 'colorlessgreenideas',
-        'username': 'noamchomsky',
-        'fullname': 'Noam Avram Chomsky',
-        'home_page': 'web.mit.edu/chomsky',
-        'description': 'Professor of Linguistics',
-        'location': 'Cambridge, MA'
+        'properties': {
```

Comment:
@csenger we decided to not wrap this data in a properties attribute by purpose. This would expose an implementation detail via API that we would like to avoid.